src/components/popup/HeadV2.tsx

```diff
@@ -110,37 +108,40 @@ export default function HeadV2({
       scrolled={scrolled}
       padding={padding}
     >
-      <BackButton
-        onClick={async () => {
-          if (back) await back();
-          else goBack();
-        }}
-      >
-        <BackButtonIcon />
-      </BackButton>
+      {showBack ? (
+        <BackButton
+          onClick={async () => {
+            if (back) await back();
+            else goBack();
+          }}
+        >
+          <BackButtonIcon />
+        </BackButton>
+      ) : null}
 
       <PageTitle>{title}</PageTitle>
 
-      <AvatarButton>
-        <ButtonAvatar
-          img={ans?.avatar || svgieAvatar}
-          onClick={() => {
-            if (!allowOpen) return;
-            setOpen(true);
-          }}
-        >
-          {!ans?.avatar && !svgieAvatar && <NoAvatarIcon />}
-          <AnimatePresence initial={false}>
-            {hardwareApi === "keystone" && (
-              <HardwareWalletIcon
-                icon={keystoneLogo}
-                color="#2161FF"
-                {...hwIconAnimateProps}
-              />
-            )}
-          </AnimatePresence>
-        </ButtonAvatar>
-      </AvatarButton>
+      {showOptions ? (
```

Comment:
This other one was probably implemented incorrectly and `allowOpen` (now removed) and `showOptions` were redundant but not implemented properly. No, the lock screen should only show the title at the top, no back button or wallet selector.